### PR TITLE
add entity-audit module

### PR DESCRIPTION
**Pull request by deepu105**
add entity-audit module
update common methods available for modules

### CODE REVIEW

File: modules/marketplace/data/modules.json [hunk 1]
```diff
@@ -89,5 +89,16 @@
...
+    {
+        "name": "Entity Audit",
+        "npmPackageName": "generator-jhipster-entity-audit",
+        "url": "https://github.com/deepu105/generator-jhipster-entity-audit",
+        "author": "Deepu KS",
+        "authorUrl": "deepu105.github.io",
+        "description": "Add support for entity audit and audit log page",
+        "keywords": ["audit", "entity", "audit log", "hibernate", "envers"],
+        "verified": false,
+        "jhVersionRequired" : "> 2.26.2"
```
Comment (jdubois) [inline modules/marketplace/data/modules.json]: Shouldn't we use the "jhi" prefix? Like elsewhere in the code?

Comment (deepu105) [inline modules/marketplace/data/modules.json]: I guess can, shall i submit another PR to correct or do you want to amend?

**Comment by jdubois** [comment]
Let's merge this first, and discuss on the second topic on the dev ML
